devito/finite_differences/operators.py

```diff
@@ -0,0 +1,77 @@
+from devito.logger import error
+
+
+def div(f):
+    """
+    Divergence of the input func.
+
+    Parameters
+    ----------
+    func: Function, VectorFunction or TensorFunction
```

Comment:
```
f : ...
```
the space ! And the name

Reply:
same story below

Reply:
if VectorFunction is a subclass of TensorFunction (or the other way around) then you just need to report the superclass in the docstring